math.java

```diff
@@ -0,0 +1,7 @@
+public class math {
+    public static int add(int num1, int num2) {
+        int answer = 0;
+        int answer = num1 + num2;
+        return ans;
```

Comment:
change the variable name to result
```suggestion
        int result = num1 + num2;
        return result;
```